website/templates/log_list.mako

```diff
@@ -55,9 +55,19 @@
                                     </span>
                                 </span>
                                 <!-- Log actions are the same as their template name -->
-                                <span data-bind="template: {name: log.action, data: log}"></span>
+                                    <!-- Handle specific language for contributor bibliographic/non-bibliographic on anonymous VOLs -->
+                                    <!-- ko if: (log.action == 'made_contributor_visible' && log.anonymous -->
```

Comment:
@sloria I think the errant parentheses on this line caused the problem. Fixing it now and then sending a new PR.